### ISSUE TITLE
Handle automatically created ES index in a more robust way when reindexing

### ISSUE THREAD

**Issue by diox** [mozilla/addons#1893]
Something we noticed while switching ES clusters today, that I think also happened the last time and we forgot to file an issue about it: there is a possible race condition where the index is automatically created by some write from another task before the reindex has time to set the index and aliases up, which makes the reindex command fail.

Using `--wipe` helps, but if we're unlucky an automatic index can still be created between the deletion and the aliases creation step.

We need to investigate if we can completely avoid automatic index creation, this is never a behavior we want, but in addition, look into these ideas:

* When `--wipe` is passed, we could create the alias sooner, before everything is indexed, as soon as the mapping is ready on the real timestamped index
* When updating the aliases, either prompt or automatically overwrite the index if it exists already: as mentioned above, we never want this index anyway, it's always wrong. And if we got that far, updating the aliases is the last step, might as well unblock it instead of failing.

It's likely only going to be a problem when we switch clusters, which doesn't happen often, but it's annoying enough that we should fix it...

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/ADDSRV-110)

**Comment by eviljeff** [comment]
I don't know how often we switch clusters but it's been 3 years... so closing.
